Gems/Atom/RPI/Code/Source/RPI.Public/Culling.cpp

```diff
@@ -255,6 +258,12 @@ namespace AZ
                             continue;
                         }
                     }
+                    else if (worklistData->m_hasExcludeFrustum && ShapeIntersection::Classify(worklistData->m_excludeFrustum, c->m_cullData.m_boundingSphere) == IntersectResult::Interior)
+                    {
+                        // Only check the exclusion frustum against entries in nodes that are contained. In the case of shadow cascades, it's
```

Comment:
Why is it unlikely? Imagine a node that is larger than main frustum, where the frustum itself is contained in the node. Then the node is not contained by the include frustum, but also any of its entries could be inside the exclude frustum. Might be good to verify that case with a unit test.

I think we need to always check the exclude frustum, if it it exists.

Reply:
You made a good point offline, this behavior is conservative, and would at worst issue draw calls for some things that don't really need to be drawn. I'm changing my blocking comment to a ship-it based on that.

Reply:
We chatted about this outside the PR, but just as an update to anyone else, The way I was doing it could create false positives (rendering things that didn't necessarily need to be) but wouldn't cause any visual artifacts. However, I decided to test out my theory and it turns out the testing _all_ the nodes is probably worth it since it reduced draw calls by another 10-15%.